config/sync/upd_featured_case_studies.settings.yml

```diff
@@ -0,0 +1,5 @@
+featured_case_study1: '25'
```

Comment:
detail, but could those be featured_case_study_1, featured_case_study_2, ... for readability.